pkg/inframetadata/internal/hostmap/constants.go

```diff
@@ -0,0 +1,41 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023-present Datadog, Inc.
+
+package hostmap
+
+import (
+	conventions "go.opentelemetry.io/collector/semconv/v1.18.0"
+)
+
+// Custom attributes not in the OpenTelemetry specification
+const (
+	attributeKernelName    = "os.kernel.name"
+	attributeKernelRelease = "os.kernel.release"
+	attributeKernelVersion = "os.kernel.version"
+)
+
+// gohai fieldPlatform subsection fields.
```

Comment:
Please extend the "gohai fieldPlatform" thing. Maybe you mean to say:
```go
// This set of constants represent fields in the Gohai payload's Platform field.
```

Reply:
Done